lib/slimmer/processors/tag_mover.rb

```diff
@@ -1,7 +1,7 @@
 module Slimmer::Processors
   class TagMover
     def filter(src, dest)
-      move_tags(src, dest, "script", dest_node: "body", keys: %w[src inner_html])
+      move_tags(src, dest, "script", dest_node: "body", keys: %w[src inner_html], scripts_for_head: %w[async defer])
```

Comment:
```suggestion
      move_tags(src, dest, "script", dest_node: "body", keys: %w[src inner_html], ignore_if_attributes: %w[async defer])
```

I was a bit confused by the name `scripts_for_head` and thought it implied the script tags were still hoisted up the head. I've suggested an alternative name, although very much open to other suggestions as well.

Reply:
ah, thanks! that's left over from an earlier iteration where they were being moved to the head, apologies.

Reply:
Following further discussion, these tags _are_ to be hoisted to the head, so this name seems appropriate!